docs/references/tanstack-start/overview.mdx

```diff
@@ -12,23 +12,13 @@ Clerk makes it simple to add authentication to your TanStack Start application.
 
 ## Client-side helpers
 
-Because Clerk TanStack Start is a wrapper around Clerk React, you can utilize the hooks that Clerk React provides. These hooks give you access to the [`Clerk`](/docs/references/javascript/clerk/clerk) object, and a set of useful helper methods for signing in and signing up. You can learn more about these hooks [in the React SDK reference](/docs/references/react/overview).
```

Comment:
same note about keeping the context

Reply:
https://github.com/clerk/clerk-docs/pull/1815/commits/a8a1b5b1b89e127b992bb55417394db4181c6afc